doc/orchestrator.rst

```diff
@@ -165,3 +165,399 @@ A full example of configuring KeyDB to run in SmartSim is shown below
   export REDIS_CONF=/path/to/keydb.conf
 
   # run smartsim workload
+
+Multiple Orchestrators
+======================
+
+SmartSim offers functionality to automate the deployment of multiple
+databases on an HPC cluster, supporting workloads that require multiple
+``Orchestrators`` for a ``Experiment``. For instance, a workload may consist of a
+simulation with high inference performance demands, along with an analysis and
+visualization workflow connected to the simulation. In such cases, SmartSim
+allows you to augment the number of clustered orchestrators or provision multiple
+co-located orchestrators, ensuring that the workload can scale to meet its resource
+requirements. Below is a general example demonstrating the process of launching
+multiple databases with unique identifiers and distributing data between them.
+
+.. note::
+
+  To run the following example, configure your environment by following
+  :ref:`Site Installation <site_installation>` then build SmartSim
+  by following the :ref:`Build Steps <from-pypi>`.
+
+Workflow Initialization and Multi Orchestrator Setup
+----------------------------------------------------
+
+Defining workflow stages requires the utilization of functions associated
+with the ``Experiment`` object. The Experiment object is intended to be instantiated
+once and utilized throughout the workflow runtime. Begin by creating a ``Experiment``
+object and assign it to the ``exp`` variable. In this example, we instantiate the
+``Experiment`` object with the name ``getting-started-multidb``. We setup the SmartSim
+``logger`` as well for use within the Experiment. For more information on the SmartSim,
+click here.
+
+.. code-block:: python
+
+  from smartsim import Experiment
+  from smartredis import Client
+  from smartredis.log import get_logger
+  import numpy as np
+
+  exp = Experiment("getting-started-multidb", launcher="auto")
+  logger = get_logger("Multidb Experiment Log")
+
+In the context of this ``Experiment``, it's essential to create and launch
+the databases as a preliminary step before any other components. We aim
+to showcase the multi-database automation capabilities of SmartSim, so we
+create two types of Orchestrators in the workflow: a co-located Orchestrator and a
+clustered Orchestrator.
+
+To create an Orchestrator, utilize the ``create_database()`` function located
+in the ``Experiment`` class. When launching more than one Orchestrator, the
+function requires specifying a unique database identifier
+argument named ``db_identifier``. The ``db_identifier``
+serves as a unique identifier for each database being launched. Usage of the ``db_identifier``
+is show in the example below.
+
+Here's how to create the databases:
+
+For the co-located database:
+
+.. code-block:: python
+
+  colocated_orch = exp.create_database(port=6379, db_nodes=1, interface="ib0", db_identifier="single_shard_db_identifier")
```

Comment:
Let's not specify the interface here. Make it as simple as possible. Also if running on a workload manager we don't need to specify a different port